OrbitBase/include/OrbitBase/Logging.h

```diff
@@ -84,6 +84,28 @@ constexpr const char* kLogTimeFormat = "%Y-%m-%dT%H:%M:%E6S";
   } while (0 && (assertion))
 #endif
 
+#define SCOPED_TIMED_LOG(format, ...)                                                          \
```

Comment:
maybe add a brief explanation as of why we declare a class inside macro, my guess is that we want to have file:line/function of the code using this macro in the logs.

Reply:
Ah, well, honestly I didn't even think about that, I just wanted to avoid the class name to "leak" globally, but that's indeed much better as before we had `OrbitCore/ScopeTimer.cpp:40`/`60` in the logs.

Reply:
Added the comment.